### PR TITLE
#15 add help menu dropdown

### PR DESCRIPTION
**Pull request by Anto8819**
#### Preview

Screenshot 2022-08-29 at 17 57 23

### CODE REVIEW

**Comment by filippomc** [comment]
@Anto8819 can you please fix the conflict?

File: applications/areg-portal/frontend/src/components/NavBar/HelpMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          p: 1.25,
+        }}
+        id="help-button"
+        aria-controls={open ? "help-menu" : undefined}
+        aria-haspopup="true"
+        aria-expanded={open ? "true" : undefined}
+        onClick={handleClick}
+      >
+        <HelpIcon
+          sx={{
```
Comment (filippomc) [inline applications/areg-portal/frontend/src/components/NavBar/HelpMenu.tsx:37]: Do we really need all this css here?

File: applications/areg-portal/frontend/src/components/icons.tsx [hunk 2]
```diff
@@ -264,3 +264,53 @@ export const DescSortedIcon = (props: SvgIconProps) => (
     />
   </SvgIcon>
 );
+
+export const FaqIcon = (props: SvgIconProps) => (
+  <SvgIcon
+    sx={{ height: "0.75rem", width: "0.875rem " }}
```
Comment (filippomc) [inline applications/areg-portal/frontend/src/components/icons.tsx:270]: A better way to set the default dimension is to change the viewbox. Specifying size here is not ideal

File: applications/areg-portal/frontend/src/theme/Theme.tsx [hunk 3]
```diff
@@ -149,6 +155,23 @@ const theme = createTheme({
         },
       },
     },
+    MuiMenu: {
+      styleOverrides: {
+        paper: {
+          width: "18.875rem",
```
Comment (filippomc) [inline applications/areg-portal/frontend/src/theme/Theme.tsx:161]: This width is better removed: any container adapts to its content by default

**Review by filippomc — CHANGES_REQUESTED** [review]
Thanks @Anto8819 dropped a few comments to improve the styling robustness

File: applications/areg-portal/frontend/src/theme/Theme.tsx (hunk 3 above)
Comment (Anto8819) [reply applications/areg-portal/frontend/src/theme/Theme.tsx:161]: I was trying to match Figma design, where the dropdown menu is wider than the content. Is ok if it's narrower than the mock up?

Comment (filippomc) [reply applications/areg-portal/frontend/src/theme/Theme.tsx:161]: we should try to match the sense of spacing that's in the mockup, but no need to match the size. Also being this in the theme, every menu in the size would be expected to have the same size, which is not ideal in a real setup

File: applications/areg-portal/frontend/src/components/NavBar/HelpMenu.tsx (hunk 1 above)
Comment (Anto8819) [reply applications/areg-portal/frontend/src/components/NavBar/HelpMenu.tsx:37]: removed

File: applications/areg-portal/frontend/src/theme/Theme.tsx (hunk 3 above)
Comment (Anto8819) [reply applications/areg-portal/frontend/src/theme/Theme.tsx:161]: done

**Review by filippomc — APPROVED** [review]
Looks good, thanks @Anto8819!

**Pull request merged**